### DISCUSSION TITLE
Postmortem: 250% cluster-wide transaction speed improvement

### DISCUSSION THREAD

**Discussion by ayende** [ravendb/ravendb#18821] [Posts]
We published a post about a performance improvement investigation that resulted in 250% performance improvement.

You can read the [full details in the blog post](https://ravendb.net/articles/postmortem-250-cluster-wide-transaction-speed-improvement).

![image](https://github.com/ravendb/ravendb/assets/116915/26009fdb-ce04-474b-9ba1-6e9f630357b7)

**Comment by georgiosd** [comment]
Amazing! Great to see these kinds of technical posts about RavenDB vs the usual marketing posts that every db does.
